framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/KieServerInstanceImpl.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.cloud.openshift.deployment;
+
+import org.kie.cloud.api.deployment.KieServerDeployment;
+import org.kie.cloud.api.deployment.KieServerInstance;
+import org.kie.cloud.openshift.OpenShiftController;
+
+public class KieServerInstanceImpl implements KieServerInstance {
+
+    private OpenShiftController openShiftController;
+    private KieServerDeployment kieServerDeployment;
```

Comment:
I wouldn't pass here whole deployment as we need to know just a namespace in instance.